Cargo.lock

```diff
@@ -2315,6 +2569,58 @@ version = "0.13.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "d196ffc1627db18a531359249b2bf8416178d84b729f3cebeb278f285fb9b58c"
 
+[[package]]
+name = "gtk"
+version = "0.18.2"
```

Comment:
can we not pull in gtk if we don't use it? It seems that would require forking muda and hiding linux backend behind feature